sc-memory/tests/scs/tests.cmake

```
@@ -1,4 +1,9 @@
-make_tests_from_folder(${CMAKE_CURRENT_LIST_DIR}/units
-  NAME scs-tests
-  DEPENDS sc-memory
-  INCLUDES ${SC_MEMORY_SRC} ${CMAKE_CURRENT_LIST_DIR}/units)
+make_tests_from_folder(
+    ${CMAKE_CURRENT_LIST_DIR}/units
+    NAME
+    scs-tests
+    DEPENDS
+    sc-memory
+    INCLUDES
+    ${SC_MEMORY_SRC}
+    ${CMAKE_CURRENT_LIST_DIR}/units)
```

```suggestion
make_tests_from_folder(
    ${CMAKE_CURRENT_LIST_DIR}/units
    NAME scs-tests
    DEPENDS sc-memory
    INCLUDES ${SC_MEMORY_SRC} ${CMAKE_CURRENT_LIST_DIR}/units)
```

Much better, because use screen space more efficient